packages/camera/camera/android/src/main/java/io/flutter/plugins/camera/MethodCallHandlerImpl.java

```diff
@@ -339,6 +339,26 @@ public void onMethodCall(@NonNull MethodCall call, @NonNull final Result result)
           }
           break;
         }
+      case "pausePreview":
+        {
+          try {
+            camera.pausePreview();
+            result.success(null);
+          } catch (Exception e) {
+            handleException(e, result);
+          }
```

Comment:
I think this is missing some unit-tests to verify exceptions are handled correctly.